commit 18f70462c7fefe7cadcc1cd618b88de565a0f717
Author: Natanael Copa <ncopa@alpinelinux.org>
Date:   Mon Aug 9 15:56:35 2010 +0000

    mkinitfs: handle the case when modules.dep has full path
    
    we do that by simply removing the prepend string

diff --git a/mkinitfs.in b/mkinitfs.in
index c3d8354..8246d07 100755
--- a/mkinitfs.in
+++ b/mkinitfs.in
@@ -92,6 +92,10 @@ BEGIN {
 		modulesdep="modules.dep";
 	FS = ": ";
 	while ( (getline < modulesdep) > 0) {
+		if (substr($0,1,1) == "/") {
+			gsub(prepend, "", $1);
+			gsub(prepend, "", $2);
+		}
 		deps[$1] = $2;
 	}
 }
